doc/transform_methods/rbasex-math.rst

```diff
@@ -0,0 +1,810 @@
+
+.. plot:: transform_methods/rbasex-peak.py
+    :align: center
+
+This was not a problem for the reconstruction method developed in [1]_, since
+it samples these functions at each pixel, with their :math:`r = \sqrt{x^2 +
+y^2}` values not limited to integers. But expanding piecewise linear radial
+distributions over the basis of these curved :math:`p_{R;n}` might be
+problematic. However, as the green curves illustrate, even for a peak with just
+3 nonzero points, its projection is represented by linear segments
+significantly better. Therefore, for real experimental data with adequate
+sampling (peak widths > 2 pixels), the piecewise linear approximation should
+work reasonably well.
+
+
+Transform
+---------
+
+The initial 3D distribution has the form
+
+.. math::
+    I(\rho, \theta') = \sum_n I_n(\rho) \cos^n \theta',
+
+where :math:`I_n(\rho)` are the radial distributions for each angular order.
+They are represented as a linear combination of the radial basis functions:
+
+.. math::
+    I_n(\rho) = \sum_R c_{R,n} b_R(\rho).
+
+The forward Abel transform of this 3D distribution (in other words, its
+projection, or the experimentally recorded image) then has the form
+
+.. math::
+    P(r, \theta) = \sum_n P_n(\rho) \cos^n \theta,
+
+where :math:`P_n(r)` are its radial distributions for each angular order (not
+to be confused with Legendre polynomials) and are represented as linear
+combinations of the radial projected basis functions:
+
+.. math::
+    P_n(r) = \sum_R c_{R,n} p_{R;n}(r)
+
+with the same coefficients :math:`c_{R,n}`.
+
+If the radial distributions of both the initial distribution and its projection
+are sampled at integer radii, these linear combinations can be written in
+vector-matrix notation as
+
+.. math::
+    \begin{aligned}
+        I_n(\boldsymbol \rho) &= \mathbf B^{\rm T} \mathbf c_n,
+        & \mathbf B_{ij} &= b_{R=i}(\rho = j), \\
+        P_n(\mathbf r) &= \mathbf P_n^{\rm T} \mathbf c_n,
+        & (\mathbf P_n)_{ij} &= p_{R=i;n}(r = j)
+    \end{aligned}
+
+for each angular order :math:`n`.
+
+It is obvious from the definition of :math:`b_R(\rho)` that :math:`\mathbf B`
+is an identity matrix, so the expansion coefficients are simply :math:`\mathbf
+c_n = I_n(\boldsymbol \rho)`. Thus the forward and inverse Abel transforms can
+be computed as
+
+.. math::
+    \begin{aligned}
+        P_n(\mathbf r) &= \mathbf P_n^{\rm T} I_n(\boldsymbol \rho), \\
+        I_n(\boldsymbol \rho) &= \big(\mathbf P_n^{\rm T}\big)^{-1} P_n(\mathbf r)
+    \end{aligned}
+
+for each angular order separately. Since all projected basis functions satisfy
+:math:`p_{R;n}(r \geqslant R + 1) = 0` (see the plots above), the matrices
+:math:`\mathbf P_n^{\rm T}` are upper triangular, and their inversions
+:math:`\big(\mathbf P_n^{\rm T}\big)^{-1}` are also upper triangular for all
+:math:`n`, which additionally facilitates the computations. (This triangularity
+makes the inverse Abel transform similar to the “onion peeling” procedure
```

Comment:
Maybe, but which? If I refer to `onion_bordas`, then I need to explain that here I “peel” spherical shells instead of circular rings in cylindrical coordinates. And then it alludes to the “polar onion-peeling” (http://dx.doi.org/10.1063/1.3126527), but that method is actually not correct (their equation 6 is valid only for infinitely narrow radial function), although it works more or less fine at sufficiently large radii.

Reply:
Well, which "onion peeling" procedure did you have in mind when you wrote this? Just the general idea of working from the outside in?